vdr/didnuts/manager.go

```diff
@@ -383,6 +261,8 @@ func (m Manager) NewDocument(ctx context.Context, keyFlags orm.DIDKeyFlags) (*or
 		return nil, err
 	}
 
+	keyFlags := DefaultKeyFlags()
```

Comment:
do the default key flags indicate all key usages we need? I think we used to have a situation where the default key flags did not indicate one we actually needed for regular v5 Nuts use cases (keyAgreement? not sure which it was)

Reply:
these are: all keyflags